app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -30,6 +30,19 @@ const getExecutions = () => [
 
 const getTotal = () => ['statistics$executions$total'];
 
+const getTotalValue = (totalDataset, datasets, label, index) => {
```

Comment:
We can use here just the `data` property from `totalDataset`, no needed to receive the whole `totalDataset` object here.